package.json

```diff
@@ -26,8 +26,8 @@
   },
   "peerDependencies": {
     "@types/sortablejs": "^1.10.0",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
```json
"react": "^16.9.0 || ^17.0.2",
"react-dom": "^16.9.0 || ^17.0.2",
```

Reply:
@waynevanson

Reply:
Using npm v7 sets the react version to 16, not being able to maintain v@17 of project which does not specify its react-version in dependencies (component libraries).

Reply:
@owen-m1

Reply:
```suggestion
    "react-dom": ">=16.9.0",
```